Something must of off with that check. I have run it manually and it was successful:
```
$ npm run prettier-check

> @goauthentik/website-docs@0.0.0 prettier-check
> prettier --check .

Checking formatting...
All matched files use Prettier code style!
```

Also, the [failed check](https://github.com/goauthentik/authentik/actions/runs/12187405291/job/33999069769?pr=12233) shows problems with file I have not touched:
```
Checking formatting...
[warn] docs/add-secure-apps/providers/proxy/_traefik_ingress.md
[warn] Code style issues found in the above file. Run Prettier with --write to fix.
Error: Process completed with exit code 1.
```

@tanberry I'm not sure what else I can do here...

--- a/website/integrations/services/home-assistant/index.md
+++ b/website/integrations/services/home-assistant/index.md
@@ -16,6 +16,9 @@ sidebar_label: Home Assistant
 :::caution
 You might run into CSRF errors, this is caused by a technology Home-assistant uses and not authentik, see [this GitHub issue](https://github.com/goauthentik/authentik/issues/884#issuecomment-851542477).
 :::
+:::caution
+Only prefixes starting with `/auth` need to be proxied (excluding prefixes starting with `/auth/token`), see [this GitHub issue](https://github.com/BeryJu/hass-auth-header/issues/212). This can be configured in the reverse proxy (e.g. nginx, Traefik) or in authentik Provider's **Unauthorized Paths**.
+:::
 :::note
 For Home Assistant to work with authentik, a custom integration needs to be installed for Home Assistant.
 :::

